src/store/currentTodoReducer.ts

```diff
@@ -0,0 +1,58 @@
+import { Todo } from '../react-app-env';
+
+const initialState: TodoState = {
+
+  todo: null,
+};
+
+export type Todoselected = {
+  type: 'TODO_SELECT',
+  payload: Todo | null,
+};
+
+export type ResetTodo = {
+  type: 'TODO_REST',
+};
+
+export type ActionTodo = (
+  Todoselected
+  | ResetTodo
+);
+
+export type TodoState = {
+  todo: Todo | null;
+};
+
+// eslint-disable-next-line max-len
+export const userReducer = (state = initialState, action: ActionTodo): TodoState => {
```

Comment:
Don't disable linter here, let's reformat
```suggestion
export const userReducer = (
  state = initialState,
  action: ActionTodo,
): TodoState => {
```